tests/vmi_lifecycle_test.go

```diff
@@ -1606,6 +1606,37 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		})
 	})
 
+	Describe("Reset a VirtualMachineInstance", func() {
```

Comment:
Have we not agreed to place this under `tests/compute/subresource`? https://github.com/kubevirt/kubevirt/pull/13208#discussion_r1876815635